controllers/clusterresources/redisuser_controller.go

```diff
@@ -174,9 +174,9 @@ func (r *RedisUserReconciler) Reconcile(ctx context.Context, req ctrl.Request) (
 			userID := fmt.Sprintf(instaclustr.RedisUserIDFmt, clusterID, user.Namespace)
 			err = r.API.DeleteRedisUser(userID)
 			if err != nil {
-				l.Error(err, "Cannot delete Redis user", "user", user.Name)
+				l.Error(err, "Cannot delete Redis user on cluster. Cluster ID: %s.", "user", clusterID, user.Name)
```

Comment:
Cannot delete Redis user on cluster -> Cannot delete Redis user from the cluster